pyproject.toml

```diff
@@ -78,7 +78,7 @@ statsmodels = "^0.13.5"
 umap-learn = "^0.5.3"
 streamlit-plotly-events = "^0.0.6"
 llvmlite = "^0.39.1"                                     # Pinning, as lower versions conflict with other libs
-encord-active-components = "^0.0.22"
+encord-active-components = "0.0.22"
```

Comment:
https://python-poetry.org/docs/dependency-specification/

use "==0.0.22" for strict exact

Reply:
that's when using the command `add`

Reply:
not what appears in the toml

Reply:
btw, you shouldn't make modifications to the toml directly, just use the poetry CLI commands which guarantee that the state is preserved. Direct modification is what got us to this fix.